CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 -   Move all form items under a `forms` folder
 -   Upgrade storybook to v8
 
+### Added
+
+-   Handle new `layoutClassnames` property on `DataList.Container` component to simplify implementation of custom
+    `DataList` layouts
```

Comment:
This first item is pretty expressive in its intent, but the second is a little vague as to its pattern, location, utility.

Reply:
Suggestions? 🤔